libs/util/shared/src/lib/links/link-utils.ts

```diff
@@ -39,37 +48,28 @@ export const FORMATS = {
   },
   gpkg: {
     extensions: ['gpkg', 'geopackage'],
-    priority: 0,
-    color: 'var(--color-primary)',
+    priority: 6,
+    color: '#ea79ba',
     mimeTypes: ['application/geopackage+sqlite3'],
   },
-  excel: {
-    extensions: ['xls', 'xlsx', 'ms-excel', 'openxmlformats-officedocument'],
-    priority: 2,
-    color: 'var(--color-primary)',
-    mimeTypes: [
-      'application/vnd.ms-excel',
-      'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
-    ],
+  zip: {
+    extensions: ['zip'],
+    priority: 7,
+    color: '#f2bb3a',
+    mimeTypes: ['application/zip'],
   },
   pdf: {
     extensions: ['pdf'],
-    priority: 0,
-    color: 'var(--color-primary)',
+    priority: 8,
+    color: '#db544a',
     mimeTypes: ['application/vnd.ms-excel'],
   },
   jpg: {
     extensions: ['jpg', 'jpeg', 'jfif', 'pjpeg', 'pjp'],
-    priority: 0,
-    color: 'var(--color-primary)',
+    priority: 8,
```

Comment:
Shouldn't either `pdf` or `jpg` have priority `9`?

Reply:
it has not been expressed so whatever they have same prio ..